core/arch/proc_api.h

```diff
@@ -312,8 +313,14 @@ typedef enum {
  *   supported at all but in some cases 15 means a feature is not supported at
  *   all, NSFLAG.
  * - If FEAT_EQ is set then the nibble value should match exactly, otherwise reg
- * values greater than or equal to the feature nibble are considered a match. The helper
- * macro below packs feature data into 16 bits (ushort).
+ *   values greater than or equal to the feature nibble are considered a match. The helper
+ *   macro below packs feature data into 16 bits (ushort).
+ *
+ * Bit 15: Set if a nibble value of 15 indicates feature not set
```

Comment:
nit: Add trailing `.` here and lines below.  Even though not a Doxygen comment that will end up as html docs, this is still a public header presented to the user.